pkg/apis/calico/types_network.go

```diff
@@ -155,10 +156,22 @@ const (
 	AutoscalingModeClusterProportional AutoscalingMode = "cluster-proportional"
 	// AutoscalingModeVPA is a constant for vertical pod autoscaling mode.
 	AutoscalingModeVPA AutoscalingMode = "vpa"
+	// AutoscalingModeStatic is a constant for static resource allocation as autoscaling mode.
+	AutoscalingModeStatic AutoscalingMode = "static"
 )
 
 // AutoScaling defines how the calico components are automatically scaled. It allows to use either vertical pod or cluster-proportional autoscaler (default: cluster-proportional).
 type AutoScaling struct {
 	// Mode defines how the calico components are automatically scaled. It allows to use either vertical pod or cluster-proportional autoscaler (default: cluster-proportional).
```

Comment:
`static` mode should be added to this comment.

Reply:
Adapted the documentation strings of corresponding fields/types in https://github.com/gardener/gardener-extension-networking-calico/pull/464/commits/d66173fd499a37fa5a80f927d5fe0cfa0a719f2d.